src/mono/wasm/runtime/startup.ts

```diff
@@ -302,7 +321,7 @@ function finalize_startup(config: MonoConfig | MonoConfigError | undefined): voi
 
         const moduleExt = Module as DotnetModule;
 
-        if(!Module.disableDotnet6Compatibility && Module.exports){
+        if (!Module.disableDotnet6Compatibility && Module.exports){
```

Comment:
Once we're adding one space we should add both! :smile:

```suggestion
        if (!Module.disableDotnet6Compatibility && Module.exports) {
```